manifest.yml

```diff
@@ -0,0 +1,13 @@
+applications:
+- name: sdx-transform-cs
+  buildpack: python_buildpack
+  health-check-type: process
+
+  env:
+    CF_DEPLOYMENT: True
+    SDX_SEQUENCE_URL: http://sdx-sequence.apps.devtest.onsclofo.uk/sequence
+    FTP_PATH: \\\\NP3RVWAPXX370\\SDX_preprod\\
```

Comment:
This is a prepod ftp path which shouldn't be in here, so it can just be set to ./ again